src/test/java/SpringJPA/UserPageTest.java

```diff
@@ -120,4 +124,17 @@ public void testAfterTrial() throws Exception{
         this.userRepository.save(loadedUser);
         this.mockMvc.perform(get("/user")).andDo(print()).andExpect(status().isForbidden());
     }
+
+    @Test
+    @DirtiesContext
+    @WithMockUser(username="User1", roles ="TRIAL")
+    public void testTrialDisplay() throws Exception{
+        User loadedUser = userRepository.findByUsername("User1");
+        int daysFromTrialStart = 5;
+        int trialDuration = 30;
+        loadedUser.setStartTime(Instant.now().plus(daysFromTrialStart, ChronoUnit.DAYS).toEpochMilli());
+        this.userRepository.save(loadedUser);
```

Comment:
Maybe you should modify the end time rather than the start time instead?

Reply:
That also makes sense, but at the end of the day the intent is to change the duration by a measured amount and have that reflected in the frontend. Don't think it needs to be adjusted.